guides/common/modules/proc_retrieving-status-of-services.adoc

```diff
@@ -1,28 +1,35 @@
 [id="Retrieving_the_Status_of_Services_{context}"]
 = Retrieving the Status of Services
 
-.Procedure
-. In the {ProjectWebUI}, navigate to *Administer > About*.
-. On the *Smart Proxies* tab, you can view the status of all {SmartProxies}.
-. On the *Compute Resources* tab, you can view the status of attached compute resource providers.
-. In the *Backend System Status* table, you can view the status of all back-end services.
+{Project} uses a set of back-end services.
+When troubleshooting, you can check the status of {Project} services.
+
+.Web UI procedure
```

Comment:
```suggestion
.Procedure
```

Also, just FYI, there's a different name for the browser-based interface in downstream orcharhino: we call it "management UI". > That's why there's an attribute "ProjectWebUI".